code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm

```diff
@@ -24,9 +24,7 @@
 		/datum/design/autolathe/ammo/rifle_short_rubber,
 		/datum/design/autolathe/ammo/hrifle_rubber,
 		//shells
-		/datum/design/autolathe/ammo/shotgun_blanks = 0,
```

Comment:
why are you removing blanks/illumination shells?
I dislike that

Reply:
Neither of those fit the theme of the disk, being LTL, you could argue illumination shells can blind people but that's not really what it's meant to do, blanks are well blanks. Moved it to the rifle ammo disk for blanks and illumination next to flare shells on misc